File: programmers/level_1/77884.cpp
#include <string>
#include <vector>
#include <cmath>

using namespace std;

int solution(int left, int right) {
    int answer = 0;

    for (int i = left; i <= right; i++) {
        if (sqrt(i) - (int) (sqrt(i)) == 0) {
            answer -= i;
        } else {
            answer += i;
        }
    }

    return answer;
}

File: kotlin_ps/src/main/kotlin/programmers_77884/77884.kt
package programmers_77884

import kotlin.math.*

class Solution {
    fun solution(left: Int, right: Int): Int {
        var answer: Int = 0

        for (i in left..right) {
            answer += if (sqrt(i.toDouble()) - sqrt(i.toDouble()).toInt() == 0.0) {
                -i
            } else {
                i
            }
        }

        return answer
    }
}